src/LaravelDebugbar.php

```diff
@@ -6,10 +6,8 @@
 use Barryvdh\Debugbar\DataCollector\CacheCollector;
 use Barryvdh\Debugbar\DataCollector\EventCollector;
 use Barryvdh\Debugbar\DataCollector\FilesCollector;
-use Barryvdh\Debugbar\DataCollector\GateCollector;
```

Comment:
The same suggestions as above - leave the import and use in the code instead of string values